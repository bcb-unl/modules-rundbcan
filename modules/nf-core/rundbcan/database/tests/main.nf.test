nextflow_process {

    name "Test Process RUNDBCAN_DATABASE"
    script "../main.nf"
    process "RUNDBCAN_DATABASE"

    tag "modules"
    tag "modules_nfcore"
    tag "rundbcan"
    tag "rundbcan/database"


    test("rundbcan - database - basic") {
        when {
            process {
                """
                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                {assert snapshot(process.out.versions).match()}
            )
        }
    }

    test("rundbcan - database - stub") {
        options "-stub"

        when {
            process {
                """
                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match()}
            )
        }
    }
}
