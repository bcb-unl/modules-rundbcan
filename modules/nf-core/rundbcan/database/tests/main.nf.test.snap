{
    "rundbcan - database - basic": {
        "content": [
            [
                "versions.yml:md5,71e74690d2a2be34140e1593c4fde889"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.5"
        },
        "timestamp": "2025-04-21T14:16:22.203313"
    },
    "rundbcan - database - stub": {
        "content": [
            [
                "versions.yml:md5,71e74690d2a2be34140e1593c4fde889"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.5"
        },
        "timestamp": "2025-04-21T14:16:26.043587"
    }
}